.gitignore

```diff
@@ -1,2 +1,3 @@
 envrc
 out
+.env
```

Comment:
missing line-ending! :D

Reply:
Leave it to me to mess up a 3-line PR ;-)